README.md

```diff
@@ -22,7 +20,55 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+    $ psql
+
+```sql
+CREATE DATABASE storage;
```

Comment:
>why do you need a local storage at all?

It can be not only local.

>why should I care about storage as a user?

Please note it is only simple gem. Do you need ready to use product?
Also please see my comment:
https://github.com/Hawk73/multi_aggregator_gem/pull/1#discussion_r135102345

>What if I specify sqlite here?

You can if it will need.